discovery_engine_core/providers/src/newscatcher.rs

```diff
@@ -261,54 +261,56 @@ pub struct Article {
     /// While Newscatcher claims to have some sort of timezone support in their
     /// [API][<https://docs.newscatcherapi.com/api-docs/endpoints/search-news>] (via the
     /// `published_date_precision` attribute), in practice they do not seem to be supplying any
-    /// sort of timezone information. As a result, we provide NaiveDateTime for now.
+    /// sort of timezone information. As a result, we provide DateTime<Utc> for now.
```

Comment:
does anybody know how a valid newscatcher `published_date` looks like if `published_date_precision` is not `timezone unknown`? it would be nice have deserialization also if `published_date_precision is `full` or `date`, because currently this causes the deserialization of the whole `Article` to fail. see also https://docs.newscatcherapi.com/api-docs/endpoints/search-news.

Reply:
@acrrd

Reply:
does not work when `published_date_precision` is `full`? Usually all articles have it

Reply:
https://demo.newscatcherapi.com/ for `full` it looks the same.
I don't know how we got so many `timezone_unknow` in our test data

Reply:
ok, then it will work for `full` and `timezone unknown` and for both cases we just continue to assume UTC 👍 i guess `date` is an edge case which can be ignored.